webdriver/tests/classic/perform_actions/key.py

```diff
@@ -24,10 +24,17 @@ def test_no_browsing_context(session, closed_frame, key_chain):
 def test_key_down_closes_browsing_context(
     session, configuration, http_new_tab, inline, key_chain
```

Comment:
Now that a window gets opened by the script we do no longer need the `http_new_tab` fixture here and in all the other tests.

```suggestion
    session, configuration, inline, key_chain
```